.github/workflows/deploy-dry-mainnet-events.yml

```
@@ -0,0 +1,41 @@
+name: Dry-run Deploy Widgets to Mainnet
```

```suggestion
name: Deploy Widgets to Mainnet - Events
```
Just for consistency in all three workflows